pilot/pkg/model/jwks_resolver.go

```diff
@@ -197,6 +197,7 @@ func (r *JwksResolver) SetAuthenticationPolicyJwksURIs(policy *authn.Policy) err
 	for _, method := range policy.Peers {
 		switch method.GetParams().(type) {
 		case *authn.PeerAuthenticationMethod_Jwt:
+			// nolint: staticcheck
```

Comment:
why we need these changes? are they auto generated as well?

Reply:
No, it's not auto generated, the lint will fail without this.

Reply:
its because we deprecated these fields